### PR TITLE
token error fixed

### LINKED ISSUE #71: JWT has expired handling
When not used for about a week the Google id JWT expires. 
This leads to the error response:
```
data: "{"Slug":"invalid_token","Message":"JWT has expired."}"
```
This should be handled by reseting the google login via deleting it.

### PR DESCRIPTION
**Pull request by matthias-luger**
fixes #71

### CODE REVIEW

**Comment by Ekwav** [comment]
this doesn't fix #71 
I think, it has to at least delete the token from localstorage as well

File: src/components/GoogleSignIn/GoogleSignIn.tsx [hunk 1]
```diff
@@ -24,6 +24,8 @@ function GoogleSignIn(props: Props) {
         setGoogleId(response.tokenId);
         api.setGoogle(response.tokenId).then(() => {
             props.onAfterLogin();
+        }).catch(() => {
+            googleId = null;
```
Comment (Ekwav) [inline src/components/GoogleSignIn/GoogleSignIn.tsx:28]: this doesn't fix #71
I think, it has to at least delete the token from localstorage as well

**Review by Ekwav — CHANGES_REQUESTED** [review]

File: src/components/GoogleSignIn/GoogleSignIn.tsx [hunk 2]
```diff
@@ -24,6 +24,9 @@ function GoogleSignIn(props: Props) {
...
+            localStorage.removeItem("googleId");
```
Comment (Ekwav) [inline src/components/GoogleSignIn/GoogleSignIn.tsx:29]: And rerender the page

**Review by Ekwav — CHANGES_REQUESTED** [review]

**Comment by Ekwav** [comment]
You caneasily test yourself by setting "googleId" in localstorage  to some random value. (thus invalid and login should be retriggered)

File: src/components/GoogleSignIn/GoogleSignIn.tsx (hunk 2 above)
Comment (matthias-luger) [reply src/components/GoogleSignIn/GoogleSignIn.tsx:29]: The GoogleLogin-Component does rerender if the googleId is set to null.
If you refer to the parent-Component. I cant, dont want and dont need to rerender this.
The onAfterLogin-Function is never executed, so the parent knows, it hasnt logged in

**Pull request merged**